apps/src/code-studio/progressRedux.js

```diff
@@ -658,6 +661,83 @@ export const progressionsFromLevels = levels => {
   return progressions;
 };
 
+export const queryUserProgress = (userId, onComplete) => (
+  dispatch,
+  getState
+) => {
+  const state = getState().progress;
+  userProgressFromServer(state, dispatch, userId, onComplete);
+};
+
+export const userProgressFromServer = (
```

Comment:
Please put this with the other helpers: https://github.com/code-dot-org/code-dot-org/blob/ff90abb8c697c9ce7f294cd193e011f55fa3561d/apps/src/code-studio/progressRedux.js#L235 

Also, since this only needs to be exported for testing purposes, please instead export it from `__testonly__` at the bottom of the file. this isn't necessarily a rule within our code, but especially given how big this file is getting it seems simpler to follow the existing convention.